src/components/Select.js

```diff
@@ -80,12 +95,20 @@ const Select = ({ name, ...props }) => {
             required={required}
             disabled={disabled}
             width={colspan}
+            style={{ display: hidden ? 'none' : 'initial' }}
```

Comment:
Please use `visibility` (see `Input`), otherwise, you may affect the layout.

Reply:
I would say there's an argument for maybe supporting both. Because sometimes you do want the space to be reserved for a hidden item but sometimes you don't.

For example
display
![Screenshot 2020-07-24 at 5 02 53 PM](https://user-images.githubusercontent.com/56241561/88377441-c37f9a00-cdd1-11ea-9429-d01db4419715.png)
visibility
![Screenshot 2020-07-24 at 5 10 27 PM](https://user-images.githubusercontent.com/56241561/88377457-cd090200-cdd1-11ea-9ac6-04f7d0491b88.png)

I would say that the first one which uses display hidden is preferable because it's a little strange to have the dropdown only take up half the space while the one below it takes up 100% of the space.

Similarly, in the ref check form, there's this case:
![Screenshot 2020-07-24 at 5 17 53 PM](https://user-images.githubusercontent.com/56241561/88377497-e8740d00-cdd1-11ea-8260-2405943d8f3d.png)
If "others" is not selected, there's extra space between the submit button and the form fields which is a little strange.
![Screenshot 2020-07-24 at 5 17 58 PM](https://user-images.githubusercontent.com/56241561/88377561-00e42780-cdd2-11ea-99d6-e4dd680f7158.png)

What do you think?

Reply:
Cool, pls change all places to use `display`, add a TODO to use animation too.